components/blitz/src/ome/formats/importer/cli/CommandLineImporter.java

```diff
@@ -804,6 +807,12 @@ public static void main(String[] args) throws Exception {
                 outputChoice = ImportOutput.valueOf(outputArg);
                 break;
             }
+            case 26: {
+                String encrytpedArg = g.getOptarg();
```

Comment:
typo in varname

Reply:
Pushed. Don't know _how_ often I did while making these few changes.